argo_stage/schema.xml

```diff
@@ -11,55 +11,31 @@
 
-    <!-- text (_t...) -->
-    <dynamicField name="*_ti"    type="text" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_tim"   type="text" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_ts"    type="text" stored="true" indexed="false" multiValued="false"/>
-    <dynamicField name="*_tsm"   type="text" stored="true" indexed="false" multiValued="true"/>
-    <dynamicField name="*_tsi"   type="text" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_tsim"  type="text" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_tiv"   type="text" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_timv"  type="text" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tsiv"  type="text" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tsimv" type="text" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-
-    <!-- English text (_te...) -->
-    <dynamicField name="*_tei"    type="text_en" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_teim"   type="text_en" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_tes"    type="text_en" stored="true" indexed="false" multiValued="false"/>
-    <dynamicField name="*_tesm"   type="text_en" stored="true" indexed="false" multiValued="true"/>
-    <dynamicField name="*_tesi"   type="text_en" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_tesim"  type="text_en" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_teiv"   type="text_en" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_teimv"  type="text_en" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tesiv"  type="text_en" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
-    <dynamicField name="*_tesimv" type="text_en" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
-
     <!-- string (_s...) -->
-    <dynamicField name="*_si"    type="string"    stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_sim"   type="string"    stored="true" indexed="true"  multiValued="true"/>
+    <dynamicField name="*_si"    type="string"    stored="false" indexed="true"  multiValued="false"/>
+    <dynamicField name="*_sim"   type="string"    stored="false" indexed="true"  multiValued="true"/>
```

Comment:
once upon a time (some years ago), Chris B. made all our fields stored.  He may have been thinking about Argo doing partial document updates, for which all fields must be stored.  However, we never ended up doing partial document updates for Argo.

If we were going to keep the stored=true values, then we have dynamicField suffixes that duplicate each other.